### PR TITLE
added missing constant

### PR DESCRIPTION
**Pull request by ncave**

### CODE REVIEW

**Comment by KevinRansom** [comment]
@ncave:

  D:\j\workspace\debug_windows---2f360ea4\src\fsharp\symbols\Exprs.fs(972,15): error FS0026: This rule will never be matched 

Apparently the match is now exhaustive :-)

**Comment by KevinRansom** [comment]
@ncave 

Thank you ... nice.

File: src/fsharp/symbols/Exprs.fs [hunk 1]
```diff
@@ -959,17 +959,16 @@ module FSharpExprConvert =
...
             | Const.UInt32  i ->  E.Const(box i, tyR)
             | Const.Int64   i ->  E.Const(box i, tyR)
-            | Const.IntPtr   i ->  E.Const(box (nativeint i), tyR)
             | Const.UInt64  i ->  E.Const(box i, tyR)
-            | Const.UIntPtr   i ->  E.Const(box (unativeint i), tyR)
-            | Const.Double   i ->  E.Const(box i, tyR)
-            | Const.Single i ->  E.Const(box i, tyR)
+            | Const.IntPtr  i ->  E.Const(box (nativeint i), tyR)
+            | Const.UIntPtr i ->  E.Const(box (unativeint i), tyR)
+            | Const.Decimal i ->  E.Const(box i, tyR)
```
Comment (dsyme) [inline src/fsharp/symbols/Exprs.fs]: Actually ``Decimal`` should be eliminated by ``TryEliminateDesugaredConstants``.  Did you encounter a case where it wasn't?

**Comment by dsyme** [comment]
The PR doesn't do any harm, but the newly added Decimal case won't be hit, see https://github.com/Microsoft/visualfsharp/blob/a989c2df7ba142117708def3a4be4b5c4ad7db8d/src/fsharp/TastOps.fs#L6134

**Comment by ncave** [comment]
@dsyme Correct, this is more for completeness in the pattern matching, in case previous stages did not eliminate Decimal const (e.g. as in Fable where the decimal support is still limited).

**Pull request merged**